### PR TITLE
[new release] coq-serapi (8.19.0+0.19.3)

### PR DESCRIPTION
**Pull request by ejgallego**
Serialization library and protocol for machine interaction with the Coq proof assistant

- Project page: <a href="https://github.com/ejgallego/coq-serapi">https://github.com/ejgallego/coq-serapi</a>
- Documentation: <a href="https://ejgallego.github.io/coq-serapi/">https://ejgallego.github.io/coq-serapi/</a>

##### CHANGES:

 - [test] Don't require math-comp to run genarg tests (@ejgallego, ejgallego/coq-serapi#399 ,
          fixes ejgallego/coq-serapi#395 , thanks to @SnarkBoojum for the report)

### CODE REVIEW

**Comment by ejgallego** [comment]
CI is fine except for:

- macos-homebrew-ocaml-4.14-arm64 (experimental)
  [tests (failed: Disconnected: Switch turned off)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/77ec0dfff43e2761d757c09d018a4bc256cf3132/variant/macos,macos-homebrew-ocaml-4.14-arm64,coq-serapi.8.19.0+0.19.3,tests)
- macos-homebrew-ocaml-5.1-arm64 (experimental)
  [tests (failed: Disconnected: Switch turned off)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/77ec0dfff43e2761d757c09d018a4bc256cf3132/variant/macos,macos-homebrew-ocaml-5.1-arm64,coq-serapi.8.19.0+0.19.3,tests)

File: packages/coq-serapi/coq-serapi.8.19.0+0.19.3/opam [hunk 1]
```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+maintainer:   "e@x80.org"
+homepage:     "https://github.com/ejgallego/coq-serapi"
+bug-reports:  "https://github.com/ejgallego/coq-serapi/issues"
+dev-repo:     "git+https://github.com/ejgallego/coq-serapi.git"
+license:      "LGPL-2.1-or-later"
```
Comment (raphael-proust) [inline packages/coq-serapi/coq-serapi.8.19.0+0.19.3/opam:6]: The repo's LICENSE file is complicated. Is it correct that this here package's tarball contains only LGPL.2.1+ code?

**Comment by raphael-proust** [comment]
These CI failures are acceptable.

File: packages/coq-serapi/coq-serapi.8.19.0+0.19.3/opam (hunk 1 above)
Comment (ejgallego) [reply packages/coq-serapi/coq-serapi.8.19.0+0.19.3/opam:6]: The tarball contains in addition to SerAPI code, which is LGPL2.1+:
- a modified version of JS sexp printer, which is MIT
- the test cases, which have several licenses, all open source
- the documentation, which is dual-licensed

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**